public/tutorials/cross-dom-bridge-erc20.js

```
@@ -1,93 +1,177 @@
+  const { optimismSepolia, sepolia } = viemChains;
+  const opActions = await import('@eth-optimism/viem/actions');
+  const { depositERC20, withdrawOptimismERC20 } = opActions;
+  
+  const l1Token = "0x5589BB8228C07c4e15558875fAf2B859f678d129";
+  const l2Token = "0xD08a2917653d4E460893203471f0000826fb4034";
+  const oneToken = 1000000000000000000n
+  
+  const PRIVATE_KEY = process.env.TUTORIAL_PRIVATE_KEY || '';
+  const account = privateKeyToAccount(PRIVATE_KEY);
+
+  const L1_RPC_URL = 'https://rpc.ankr.com/eth_sepolia/<YOU_API_KEY>';
```

ESLint

[error] Extra semicolon.

(semi)